weave-js/src/components/Dialog/Dialog.tsx

```diff
@@ -51,9 +51,9 @@ export const Overlay = React.forwardRef(
  */
```

Comment:
fyi @jo-fang -- I added some bare minimum styles for this specific use case. sorry if it causes conflicts for you 🙇‍♀️

Reply:
this was the only change I had planned if you want to bake it into this PR as well. ty for this

https://github.com/wandb/weave/compare/joyce/center-dialog?expand=1

Reply:
updated!